src/java/org/apache/cassandra/service/accord/txn/TxnDataName.java

```diff
@@ -221,6 +243,7 @@ public long serializedSize(TxnDataName t, int version)
             int size = Byte.BYTES + sizeofUnsignedVInt(t.parts.length);
             for (String part : t.parts)
                 size += TypeSizes.sizeof(part);
+            size += serializedNullableSize(t.clustering, version, clusteringPrefixSerializer);
```

Comment:
Aside: One of the items I have in the Alpha -> v1 TODO list is changing the internal representation of `parts` to not force converting everything to `String`